src/modules/store/modules/drawing.store.js

```diff
@@ -39,6 +39,14 @@ export default {
          */
         drawingKmlIds: null,
     },
+    getters: {
+        getDrawingPublicFileUrl: (state) => {
+            if (state.drawingKmlIds) {
+                return `https://public.geo.admin.ch/${state.drawingKmlIds.fileId}`
```

Comment:
I don't think there is a stagging (yet) for this service. But yes it might change as soon as we switch to the new `service-kml`, that might be a good idea